### PR TITLE
Add contributors section

### PR DESCRIPTION
**Pull request by dholbach**
Fixes: #349

This first draft is not working 100% yet. We will need to figure out how to add a GH token to the Netlify build. I guess another option would be to commit `content/en/contributors_include.html` and create a GH action which updates it regularly through running the `./hack/contributors.py` script and file a PR.

Thoughts?

### CODE REVIEW

**Comment by dholbach** [comment]
Firefox_Screenshot_2021-06-24T10-49-28 119Z

**Comment by dholbach** [comment]
I'd also need some help beautifying the section somewhat.

**Comment by dholbach** [comment]
Firefox_Screenshot_2021-06-24T12-50-31 147Z

File: hack/contributors.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        'image-automation-controller',
+        'image-reflector-controller',
+        'kustomize-controller',
+        'notification-controller',
+        'pkg',
+        'source-controller',
+        'source-watcher',
+        'terraform-provider-flux',
+        'website',
+        'webui'
```
Comment (dholbach) [inline hack/contributors.py:39]: Are these all the repos we want to check - or should we include *all* /fluxcd repos?

**Comment by dholbach** [comment]
Firefox_Screenshot_2021-06-24T13-18-30 435Z

**Comment by hiddeco** [comment]
I would put the contributors below the community information, as the information has more value to a visitor than the faces of all the lovely people that got us here.

**Comment by dholbach** [comment]
Done.

Now I need help hooking this up. AFAICS our options are:

1. put a GH token into Netlify somewhere, run script on every build
2. run script by a cron job in a GH Action (weekly?) which generates a PR to update `content/en/contributors_include.html`

**Comment by stefanprodan** [comment]
I would opt for a weekly job that opens a PR.

**Comment by dholbach** [comment]
Thanks @stefanprodan - will take a look!

**Comment by dholbach** [comment]
Appears to be working: https://github.com/dholbach/website/runs/2931141709

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @dholbach ☕

**Pull request merged**